src/anyio/_backends/_asyncio.py

```diff
@@ -141,7 +141,9 @@ def find_root_task() -> asyncio.Task:
         for task in all_tasks():
             if task._callbacks:
                 for cb in _get_task_callbacks(task):
-                    if cb is _run_until_complete_cb or cb.__module__ == 'uvloop.loop':
+                    if isinstance(cb, partial):
+                        continue
```

Comment:
```suggestion
```

I think the getattr is all we need here, thoughts @agronholm ?

Reply:
I pushed changes to only use the `getattr` since that seems cleaner to me.